RELEASE_NOTES.md

```diff
@@ -7,51 +7,46 @@ Example Note:
+In this release added 'daemon' option and renamed`--chef-service-interval` option to `--chef-daemon-interval`.
+
+Updated code to work with latest azure-sdk gems i.e. version 0.9.0.
 
 New options introduced:
-`--chef-service-interval`, which lets the user specify the frequency at which chef-client service should run. This option is available for both ASM and ARM commands. For ASM, it's supported with `cloud-api` bootstrap protocol.
 
-`--extended-logs` shows chef convergence logs in detail. This option is available for both ASM and ARM commands. For ASM, it's supported with `cloud-api` bootstrap protocol.
+`--azure_availability_set`, which allows user to create virtual machines in required availability set.
+
+`--daemon`, which allows user to select options to run chef-client as none, service or task. This option works for windows node and --bootstrap-protocol to be 'cloud-api'.
 
 Please file bugs or feature requests against the [KNIFE_AZURE](https://github.com/chef/knife-azure/issues) repository.
 More information on the contribution process for Chef projects can be found in the [Chef Contributions document](https://docs.chef.io/community_contributions.html).
 
 ## knife-azure on RubyGems and Github
 https://rubygems.org/gems/knife-azure
+
 https://github.com/chef/knife-azure
 
 ## Features added in this release:
 
-See the [1.7.0 CHANGELOG](https://github.com/chef/knife-azure/blob/1.7.0/CHANGELOG.md)
-for the complete list of features added in this release.
+See the [1.8.0 CHANGELOG](https://github.com/chef/knife-azure/blob/1.8.0/CHANGELOG.md) for the complete list of features added in this release.
 
 Here is a partial list:
 
-* Added chef_service_interval option and moved encrypted_data_bag_secret option from public_config to private_config for chef_extension: [\#408](https://github.com/chef/knife-azure/pull/408) ([Aliasgar16](https://github.com/Aliasgar16))
-* Added support to show extended logs in ASM bootstrap command: [\#400](https://github.com/chef/knife-azure/pull/400) ([Aliasgar16](https://github.com/Aliasgar16))
-* Added support for extended logs to knife arm bootstrap command: [\#398](https://github.com/chef/knife-azure/pull/398) ([Aliasgar16](https://github.com/Aliasgar16))
-* Add support passing of tcp port option in ARM: [\#394](https://github.com/chef/knife-azure/pull/394) ([dheerajd-msys](https://github.com/dheerajd-msys))
-* Added support for running Xplat command in case of token expiry: [\#378](https://github.com/chef/knife-azure/pull/378) ([dheerajd-msys](https://github.com/dheerajd-msys))
-* Added support for extended_logs to show chef-client run logs to user in ARM server_create command: [\#372](https://github.com/chef/knife-azure/pull/372) ([Aliasgar16](https://github.com/Aliasgar16))
+* Added `--azure_availability_set` option which allows to create virtual machine in required availability set. [\#453](https://github.com/chef/knife-azure/pull/453) ([piyushawasthi](https://github.com/piyushawasthi))
```

Comment:
*option which allows the user to create virtual machines in specified availability set.